### PR TITLE
Add migration to regrant permissions to crud user

### PR DESCRIPTION
**Pull request by gidjin**
## Summary

It was discovered recently that creating a new view in the demo environment is not being created with permissions for the MilMove database user `crud` to access it. This results in a `error pq: permission denied for view duty_stations` error. 

- [this slack thread](tbd)
- [this other thread]()
- see these migrations
  - migrations/app/schema/20210113173236_create_crud_role.up.sql
- Possibly related migrations
  - migrations/app/schema/20201230070814_fix_ecs_user_in_docker.up.sql
  - migrations/app/schema/20190129115001_master_role.up.sql
  - migrations/app/secure/20190129115416_ecs_user_for_rds.up.sql
  - migrations/app/secure/20190726151515_grant_all_ecs_user.up.sql
 
This needs to be tested in experimental or demo to find out if the migration as it is would be best or if there is something else going on that is preventing permissions from being on the view. Alternate solution that worked but would require a `GRANT` statement for every future view.

```sql
GRANT SELECT, INSERT, UPDATE, DELETE ON duty_stations TO crud;
GRANT SELECT, INSERT, UPDATE, DELETE ON move_to_gbloc TO crud;
```

## Setup to Run Your Code

I can't replicate this locally. I checked and my local db has a `crud` user but that user has the correct permissions on the view. If you can replicate it locally reseting the database in exp or demo won't be needed to experiment with it. I do have this deployed to Demo for now.

### Additional steps

1. Log into the office in demo and ensure no 500 error. 
2. Use the api to retrieve move with code `7799QT` from demo

## Verification Steps for Author

These are to be checked by the author.

- [X] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [X] Request review from a member of a different team.

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Frontend

- [ ] User facing changes have been reviewed by design.
- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
- [ ] There are no new console errors in the browser devtools
- [ ] There are no new console errors in the test output
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide#logging)
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide/#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#zero-downtime-migrations)
- [ ] Have been communicated to #g-database
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#secure-migrations)

### CODE REVIEW

**Review by ahobson — APPROVED** [review]
I reviewed the [pg docs](https://www.postgresql.org/docs/12/sql-grant.html) and I agree that this should do the trick. I wonder if when we created demo we restored from a db backup and that may not have included these role changes.

We really shouldn't be making role changes inside a migration for that reason, but that's a problem for another day.

To explain a bit more: [pg_dump](https://www.postgresql.org/docs/12/app-pgdump.html) only backs up a single database. Role level permissions, however, are at the cluster level. As that page says

> pg_dump only dumps a single database. To back up an entire cluster, or to back up global objects that are common to all databases in a cluster (such as roles and tablespaces), use pg_dumpall.

 Our migrations conflate changes to a single database with changes that affect the entire cluster. This is one reason why squashing our migrations is so difficult: we really need to separate out these role changes into a separate database configuration process.

File: .circleci/config.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ references:
   #
   # In addition, it's common practice to disable acceptance tests and
   # ignore tests for demo deploys. See the branch settings below.
-  demo-branch: &demo-branch placeholder_branch_name
+  demo-branch: &demo-branch gidjin_fix_duty_station_view_permissions
```
Comment (ahobson) [inline .circleci/config.yml:29]: Do we want these branch changes to go into master? I don't think we do

Comment (gidjin) [reply .circleci/config.yml:29]: No this was to get the fix onto Demo for Dan to check out. I'll remove them before merging.

**Comment by ahobson** [comment]
Thank you! Please merge when ready!

**Pull request merged**